FATN_to_USB/fatn_to_usb/main.py

```
@@ -0,0 +1,370 @@
+def import_display_module():
+    """
+    Import and initialize display module using detection system.
+
+    Returns:
+        tuple: (display object, do_mes function) or (None, None) if unavailable
+    """
+    try:
+        from scrolling_text import do_mes, disp
+        # Display is already initialized by scrolling_text module
+        # which uses the display_detector system
+        return disp, do_mes
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)